src/nspawn/nspawn-settings.c

```diff
@@ -202,7 +202,7 @@ int config_parse_capability(
                         continue;
                 }
 
-                u |= 1 << ((uint64_t) cap);
+                u |= 1ULL << ((uint64_t) cap);
```

Comment:
This *is* correct, but I think we can make it more direct and simpler. `u` is defined as `uint64_t`, so let's use a `uint64_t` constant too. Then the advantage is that the cast is not necessary anymore:
```C
u |= UINT64_C(1) << cap;
```